src/Features/RulesMissingDocumentation.md

```diff
@@ -2,27 +2,24 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
+IDE0005_gen |  | Imports statement is unnecessary. |
 IDE0043 |  | Invalid format string |
-IDE0120 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0120> | Simplify LINQ expression |
-IDE0130 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0130> | Namespace does not match folder structure |
-IDE0150 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0150> | Prefer 'null' check over type check |
-IDE0160 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0160> | Convert to block scoped namespace |
-IDE0161 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0161> | Convert to file-scoped namespace |
-IDE0170 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0170> | Property pattern can be simplified |
-IDE0180 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0180> | Use tuple to swap values |
-IDE0200 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0200> | Remove unnecessary lambda expression |
```

Comment:
Something is wrong. IDE0200 was here when I initially opened the PR. Now when I re-ran the tool, it's gone and it's still not documented. Not sure what's special with IDE0200 specifically.